BlueprintUICommonControls/Sources/AccessibilityElement.swift

```diff
@@ -3,6 +3,24 @@ import UIKit
 
 public struct AccessibilityElement: Element {
 
+    /// Used to provide addtional functionality to assistive technogies beyond your accessible UI.
+    public struct CustomAction {
+        public typealias OnActivation = (String) -> Bool
+
+        /// A localized name that discribes the action.
+        public var name: String
+        /// An image representing the action to be shown with some assistive technologies such as Switch Control.
+        public var image: UIImage?
+        /// A Callback for when the action is activated. The action's name will be passed as a parameter. This should return a `bool` indicating success or failure of the action.
```

Comment:
Why do we need to pass the name in here? Isn't it declared alongside a name?

Reply:
I guess we don't need to, Apple's implementation passes the whole object back, so i was just extracting the relevant bits from that but you're right, it can pretty much be ignored.